### PR TITLE
Feature/barometer

### PR DESCRIPTION
**Pull request by masonturner**

### CODE REVIEW

**Comment by jpanikulam** [comment]
Could you add a file in `embedded` that uses this?

**Comment by masonturner** [comment]
> Could you add a file in `embedded` that uses this?
Besides the `bmp388_test.cc` file? Or should I move that file to embedded... yeah, I'll do that.

File: third_party/bmp388/Adafruit_BMP3XX.cc [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+  the_sensor.intf = BMP3_I2C_INTF;
+  the_sensor.delay_ms = delay_msec;
+  the_sensor.i2c_device = _i2cDevice;
+
+  int8_t rslt = BMP3_OK;
+  rslt = bmp3_init(&the_sensor);
+
+#ifdef BMP3XX_DEBUG
+  std::cout << "Result: " << rslt << std::endl;
+#endif
```
[the remark this answers is not in the document]
Comment (masonturner) [reply third_party/bmp388/Adafruit_BMP3XX.cc]: I changed it to `std::cout`.  I think it was previously a `Serial.println()` call.

File: third_party/CMakeLists.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 add_subdirectory(i2c)
-add_subdirectory(bno055)
\ No newline at end of file
+add_subdirectory(bno055)
+add_subdirectory(bmp388)
```
[the remark this answers is not in the document]
Comment (masonturner) [reply third_party/CMakeLists.txt]: Done.

**Comment by jpanikulam** [comment]
Awesome, merging when it compiles.

**Pull request merged**